install.sh

```diff
@@ -3,56 +3,81 @@
 # Exit on fail.
 set -e
 
-RESET='\e[0m'
+# Default chromebrew repo values.
+: "${GTIHUB_OWNER:=chromebrew}"
+: "${GITHUB_REPO:=chromebrew}"
+: "${GITHUB_BRANCH:=master}"
+: "${GITLAB_REPO:=26210301}"
```

Comment:
To be honest, I don't know if there's any practical value to changing the gitlab repo, I don't know of any other repos that mirror our package repository or would be otherwise useful to switch to.

Reply:
This is used to set the `PREBUILT_URL` variable below to reduce duplicate long assignments.  This is actually good because if this value changes in the future, we only have to change in one place...here.

Reply:
Wouldn't that be the same if we just had the value set in PREBUILT_URL? I'm not a fan of single-use variables.